PlanarAlly/planarserver.py

```diff
@@ -871,9 +875,7 @@ def shape_wrap(player, shape):
 app.on_shutdown.append(on_shutdown)
 
 if __name__ == '__main__':
-    app['background_save'] = sio.start_background_task(save_all)
-    cfg = configparser.ConfigParser()
-    cfg.read("server_config.cfg")
+    sio.start_background_task(save_all)
```

Comment:
You want to keep the app['background_save'] assignment, as it is used on line 84 to cancel the task.

Reply:
Done that!

Reply:
Now you start the background save twice ;)

Reply:
I have updated it, do you mean like that?

Reply:
indeed